Jamulus.pro

```diff
@@ -1120,6 +1120,6 @@ contains(CONFIG, "disable_version_check") {
 # be sure to update .github/workflows/coding-style-check.yml and .clang-format-ignore as well.
 CLANG_FORMAT_SOURCES = $$files(*.cpp, true) $$files(*.mm, true) $$files(*.h, true)
 CLANG_FORMAT_SOURCES = $$find(CLANG_FORMAT_SOURCES, ^\(android|ios|mac|linux|src|windows\)/)
-CLANG_FORMAT_SOURCES ~= s!^\(windows/\(nsProcess|ASIOSDK2\)/|src/res/qrc_resources\.cpp\)\S*$!!g
+CLANG_FORMAT_SOURCES ~= s!^\(windows/nsProcess/|libs/ASIOSDK2|src/res/qrc_resources\.cpp\)\S*$!!g
```

Comment:
@hoffie I think you should check if this is ok here.

Reply:
I think line 1122 could change to `CLANG_FORMAT_SOURCES = $$find(CLANG_FORMAT_SOURCES, ^\(src)/)`
since all sources are under src now ?

Reply:
No. There's still some cpp code in these folders(androiddebug.cpp) for example

Reply:
Wouldn't it be logical to move those somewhere under src too ? (perhaps libs/android?)

Reply:
> Wouldn't it be logical to move those somewhere under src too ?

Yes, maybe, but probably not as part of this PR which is titled and branch-named to touch sound files. :)

> (perhaps libs/android?)

It's not a generic lib, rather a platform-specific extension. Also, `libs/` is used for third-party imported code only and it has benefits to keep it that way. As these are not many files, it might be ok to put them directly under src/ and prefix (mac_activity.mm) or suffix (activity_mac.mm) them accordingly. Again... different PR. :)

@ann0see Looks good at first glance, you can run qmake and check the Makefile's clang_format target whether it includes ASIOSDK2 or not (it obviously should not).

Reply:
I checked it yesterday and it seems to be ok.